src/services/plugins/socket.rs

```diff
@@ -0,0 +1,220 @@
+use crate::{jobs::JobProducerTrait, models::AppState};
```

Comment:
would be great to add header comments to a new files that are missing it.
This improves our rust technical documentation a lot